ecommerce-product-page-main/styled.css

```diff
@@ -200,98 +197,65 @@ body{
     background-color: hsl(26,100%,55%);
 }
 
-
-
-img.space{
-    margin-top: 12px;
-    width: 130px;
-}
-
-ul.list{
-
+.list {
     display: flex;
-    list-style: none;
+    text-decoration: none;
+    gap: 10px;
+    align-items: center;
 }
 
 
-
-ul.list li+li:before{

```

Comment:
Do you think the way i directed/arranged my class here is inaccurate or just a longer route to take? cos i noticed you used another css route here.